BD-3440 Add Trending item recommendations (#8039)
braze-inc/braze-docs: _docs/_user_guide/sage_ai/recommendations/about_item_recommendations.md

{% enddetails %}

### Trending purchased items

Highlight items that your users have recently purchased with increased frequency. For example, an ecommerce business could recommend seasonal items that users are starting to stock up on during their preparations for the next season. 

{% details Requirements %}

> - AI item recommendations
> - Catalog of relevant items
> - A method for tracking purchase (either a purchase object or custom event)

Reviewer: ```suggestion
- AI item recommendations
- Catalog of relevant items
- A method for tracking purchases (either a purchase object or custom event)
```

{% enddetails %}

{% details How to set up %}

1. Create an [AI item recommendation]({{site.baseurl}}/ai_item_recommendations/).
2. Set the **Type** to **Trending**.
3. Select your catalog.
4. (Optional) Add a selection to filter your recommendation to only relevant items.
5. Choose either a purchase event or a custom event that tracks purchases, along with the corresponding property.
6. Train the recommendation.
7. [Use the recommendation in messaging.]({{site.baseurl}}/user_guide/sage_ai/recommendations/ai_item_recommendations/#using-recommendations-in-messaging)

{% enddetails %}

### Trending liked items

> Highlight items that your users have recently liked with increased frequency. For example, a music app could feature up-and-coming artists who've had a recent surge in user likes.

Reviewer: ```suggestion
Highlight items that your users have recently liked with increased frequency. For example, a music app could feature up-and-coming artists who have experienced a recent surge in user likes.
```
